packages/tables/src/styled/StyledRow.ts

```diff
@@ -23,36 +23,79 @@ export interface IStyledRowProps {
   size?: ITableProps['size'];
 }
 
+const baseRowColorStyles = ({ theme, isStriped }: IStyledRowProps & ThemeProps<DefaultTheme>) => {
+  return css`
+    border-bottom: ${theme.borders.sm} ${getColor({ variable: 'border.subtle', theme })};
+    background-color: ${isStriped &&
+    getColor({
+      variable: 'background.emphasis',
+      transparency: theme.opacity[100],
+      light: { offset: -300 },
+      theme
+    })};
```

Comment:
Given the context and how `background.subtle` is used in other components, wouldn't `background.subtle` fit better semantically?

```suggestion
    getColor({
      variable: 'background.subtle',
      transparency: theme.opacity[100],
      light: { offset: 300 },
      dark: { offset: -600 },
      theme
    })};
```

Reply:
I would be good to understand Design's intent for using a custom color / opacity combo.

With Chrome's color-picker tool, I got the "flattened" HEX value for the stripped row background color. That value corresponds to the color a visual user perceives when the row color (neutral (400:600)/0.08%) is applied over our default light / dark background.

Using this [tool](https://colordesigner.io/compare-colors) I compared their DeltaE:


### Light

custom: #f9f9fa
`background.subtle`: #f8f9f9

![Screenshot 2024-06-06 at 10 18 22 AM](https://github.com/zendeskgarden/react-components/assets/6879688/e72a58a0-051d-4bab-a70b-5519608ffd9d)


### Dark

custom: #1e2328
`background.subtle`: #1c2227

![Screenshot 2024-06-06 at 10 19 19 AM](https://github.com/zendeskgarden/react-components/assets/6879688/30c22c71-f4cf-4d22-81c7-132c7817ac07)

As you can see, the colors are nearly identical.


> Typically, a Delta E of less than 1 is not perceptible to the human eye.


If `Table` is always rendered on top of a backdrop with a `background.default`, would it make more sense to reuse `background.subtle`?

Reply:
And thank you for the detailed breakdown / comparison with other components in this use case. TIL!

Reply:
Something I'm thinking about is what it means for the color and its semantics when we need to do a 6 level offset (plus/minus `600`). Does the color respect its design intent by the time its shade is offset by this much? @lucijanblagonic perhaps a question for you.

Reply:
The backgrounds of both GroupRow and Row (when striped) are set to the below value, per Flo's suggestion, which is the reason for my inquiry/curiosity around color intent with major offsets:

```
 getColor({
      variable: 'background.subtle',
      transparency: theme.opacity[100],
      light: { offset: 300 },
      dark: { offset: -600 },
      theme
})
```

Reply:
> That value corresponds to the color a visual user perceives when the row color (neutral (400:600)/0.08%) is applied over our default light / dark background.

that is exactly the intent! we wanted a value that visually matches the v8 group/striped bg treatment. i've attached an image to visually describe the points below:

![image](https://github.com/zendeskgarden/react-components/assets/39511661/16721f5c-ad67-4a88-be49-024cdf1b1722)

1. at first glance, `background.subtle` was the first logical replacement, but that is intentionally a solid fill, and a solid fill would not respect the bg subtle, raised, and recessed treatments in both light and dark modes.
2. with that considered we opted for the existing neutral opacity, which targets 700 and 500 @ .8, but in light mode it was too dark of a bg (albeit dark mode bg was just fine)
3. then opted for the grey-400 / grey-600 @ .08 as it was the closest match.

if there's a more efficient way to obtain that visual parity, then i'm all for it!